src/Microsoft.TestPlatform.Build/Resources/Resources.resx

```diff
@@ -129,6 +129,9 @@
   <data name="TestRunningSummary" xml:space="preserve">
     <value>Test run for {0}({1})</value>
   </data>
+  <data name="UpdateTemplateForCollectingXPlatCodeCoverage" xml:space="preserve">
+    <value>Warning: Update the test project template and add reference to Microsoft.TestPlatform.Extensions.CoverletCoverageDataCollector to collect xplat code coverage</value>
```

Comment:
What do we mean by updating the Test project template ?

Reply:
We will be adding the datacollector in csproj like `<PackageReference Include="Microsoft.TestPlatform.Extensions.CoverletCoverageDataCollector" Version="1.0.0" />`
This meant adding the reference.

Reply:
it seems redundant then,
Change it to "To collect xplat code coverage please add reference to Microsoft.TestPlatform.Extensions.CoverletCoverageDataCollector to the test project."

And we can add more information and a link we plan to blog it.